### PR TITLE
Complete the homework №7

### PR DESCRIPTION
**Pull request by natak8200**

### CODE REVIEW

File: src/main/java/hw_7/Figures/Circle.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    @Override
+    public double getArea() {
+        return 2 * radius * PI;
+    }
+
+    @Override
+    public String toString() {
+        return "Circle" + super.toString();
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_7/Figures/Circle.java:24]: Please remove unused methods

File: src/main/java/hw_7/climate_control/Climate.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void cool_down() {
+        System.out.println("Climate is cool down the room");
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_7/climate_control/Climate.java:22]: Please remove unused methods and constructor

File: src/main/java/hw_7/climate_control/Conditioner.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    @Override
+    public String toString() {
+        return "Conditioner{" +
+                "model=" + getModel() +
+                ",color=" + getColor() +
+                ",freonName='" + freonName + '\'' +
+                '}';
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_7/climate_control/Conditioner.java:23]: Please remove unused methods

File: src/main/java/hw_7/climate_control/Fan.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        System.out.println("The fan can switch speed settings for cooling");
+    }
+
+    @Override
+    public String toString() {
+        return "Fan{" +
+                "model=" + getModel() +
+                ", color=" + getColor() +
+                ", number_of_speed=" + number_of_speed +
+                '}';
```
Comment (AntBelyi) [inline src/main/java/hw_7/climate_control/Fan.java:22]: Please remove unused methods

File: src/main/java/hw_7/climate_control/Humidifier.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    public void setWater_quantity(int water_quantity) {
+        this.water_quantity = water_quantity;
+    }
+
+    @Override
+    public void cool_down() {
+        System.out.println("Works as a cooler and humidifier for water evaporation");
+    }
```
Comment (AntBelyi) [inline src/main/java/hw_7/climate_control/Humidifier.java:14]: Please remove unused methods

**Comment by natak8200** [comment]
fixed

File: src/main/java/hw_7/Figures/Circle.java (hunk 1 above)
Comment (natak8200) [reply src/main/java/hw_7/Figures/Circle.java:24]: fixed

File: src/main/java/hw_7/climate_control/Climate.java (hunk 2 above)
Comment (natak8200) [reply src/main/java/hw_7/climate_control/Climate.java:22]: fixed

File: src/main/java/hw_7/climate_control/Conditioner.java (hunk 3 above)
Comment (natak8200) [reply src/main/java/hw_7/climate_control/Conditioner.java:23]: fixed

File: src/main/java/hw_7/climate_control/Fan.java (hunk 4 above)
Comment (natak8200) [reply src/main/java/hw_7/climate_control/Fan.java:22]: fixed

File: src/main/java/hw_7/climate_control/Humidifier.java (hunk 5 above)
Comment (natak8200) [reply src/main/java/hw_7/climate_control/Humidifier.java:14]: fixed

**Pull request merged**